client/components/popover/menu.jsx

```diff
@@ -20,6 +21,7 @@ var PopoverMenu = React.createClass( {
 
 	getDefaultProps: function() {
 		return {
+			autoPosition: true,
```

Comment:
Let's set the default to `false`. We can run into strange behavior if there's not enough room available. cc @retrofox  for help testing